src/qt/openuridialog.h

```diff
@@ -7,6 +7,8 @@
 
 #include <QDialog>
 
+#include <qt/platformstyle.h>
```

Comment:
Just forward declare `PlatformStyle`.

Reply:
@promag 

> Honestly I don't see value in the shortcut, I'd remove it from the others and make sure the line edit has focus.

The line has focus. I copied the shortcut from the other occurrences of this button

> I kind of dislike automatic connections since it's easy to break them - for instance renaming the item.

Renaming them would cause an error

Reply:
What error? The doc at https://doc.qt.io/qt-5/qmetaobject.html#connectSlotsByName doesn't mention it.

Reply:
Nevermind, I only changed the occurrence in the .cpp file and not the qt file